src/types.h

```diff
@@ -9,10 +9,15 @@
 #ifndef LIBREALSENSE_TYPES_H
 #define LIBREALSENSE_TYPES_H
 
+// Disable declspec(dllexport) warnings:
```

Comment:
Pls put decorator to limit the scope
 #pragma warning( push )
 #pragma warning( disable:  )
 //code
 #pragma warning( pop